entries/entry01.md

```diff
@@ -1,7 +1,17 @@
 # Entry 1
```

Comment:
So cool! But missing sources, EDP, skills. Not taking off for professionalism, but try to include more markdown syntax next time.